app/models.py

```diff
@@ -1614,6 +1614,75 @@ def serialize(self):
         }
 
 
+class DirectAwardProject(db.Model):
```

Comment:
So I was wanting to put these into an `app.models.direct_award` module. Then this model would just be called `Project`.

Which would mean moving all the existing code in `app.models` into a new `__init__.py`, which is a pain... @allait if you don't agree then we can just move on?